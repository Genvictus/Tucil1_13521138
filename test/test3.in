A 4 J 4