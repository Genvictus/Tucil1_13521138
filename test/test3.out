(4 * (11 - (1 + 4)))
(4 * ((11 - 1) - 4))
(4 * (11 - (4 + 1)))
(4 * ((11 - 4) - 1))
((11 - (1 + 4)) * 4)
(((11 - 1) - 4) * 4)
((11 - (4 + 1)) * 4)
(((11 - 4) - 1) * 4)
